requirements/requirements.txt

```diff
@@ -1,6 +1,10 @@
-pexpect~=4.6
+pexpect~=4.9
 requests~=2.26
 json_database~=0.7
 kthread~=0.2
 watchdog
 pyee
+combo-lock~=0.2
+rich-click~=1.7
+rich~=13.7
+orjson
```

Comment:
This should probably pin a major version to prevent resolving a breaking new version